test/e2e/networking.go

```diff
@@ -267,15 +325,17 @@ func LaunchNetTestPodPerNode(f *Framework, nodes *api.NodeList, name, version st
 			Spec: api.PodSpec{
 				Containers: []api.Container{
 					{
-						Name:  "webserver",
+						Name: "webserver",
+						//versions ~ 1.3 (original RO test), 1.6 uses newer services/tokens,...
```

Comment:
maybe include this comment up at the top next to `nettestVersion`.

or if we're not likely to be changing it dynamically, it's not even clear this needs to be a parameter passed to `launchNetTestPodsPerNode`.